src/features/onboarding/LinkAccount.tsx

```diff
@@ -30,7 +30,7 @@ const LinkAccount = () => {
         } else {
           Alert.alert(
             'Helium App Not Found',
-            'You must have the official Helium app installed.',
+            'You must have the Helium app installed from official App Store.',
```

Comment:
You must have the [Helium wallet app](https://docs.helium.com/wallets/app-wallet/) installed using the official Android Play Store or iOS App Store. @posterzh suggested improvement